CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.0-rc.7 (2018-12-13)
+
+- Added option to take action before modals are closed and prevent them from closing. [#7](https://github.com/blackbaud/skyux-modals/pull/7)
+- Fixed section forms in modals to set minimum height of 460px. [#18](https://github.com/blackbaud/skyux-modals/pull/18)
+- Fixed modal service to stop throwing errors during initialization. [#13](https://github.com/blackbaud/skyux-modals/pull/13)
```

Comment:
This is really minor, but I think "to prevent errors" sounds a lot better than "to stop throwing errors."

Reply:
done